morph/react-dom/block-views-flow.js

```diff
@@ -0,0 +1,12 @@
+export function enter(node, parent, state) {
+  if (parent || !node.isView || !node.isDesignSystemRoot) return
+
+  state.render.push('<fromFlow.ViewsFlow viewPath={viewPath}>')
```

Comment:
I see that you redefine `viewPath` below. Don't we want the `viewPath` here to refer to the actual place in which the DS view is being mounted on which is passed from the props?

Reply:
you are right, I'll fix it to pass the parent viewPath here. thanks